workflow/apm/services.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: APM Dashboard - Services
+sidebarTitle: Services
+---
+
+The APM Services Dashboard displays all of your APM services in a unified dashboard. A service groups together interconnected endpoints, queries, or jobs within your application. This is useful for contextualizing broader issues within your software infrastructure. 
```

Comment:
Generally I stay away from the word Dashboard because we have a dashboard buidler which is where that term is going to mostly be used. Instead I would recommend calling this pages or views (page for the navigation element and view for a collection of data). There's also too much use of the word APM and services in this section.